x-pack/test/plugin_api_integration/test_suites/task_manager/task_manager_integration.js

```diff
@@ -420,6 +419,8 @@ export default function ({ getService }) {
         expectReschedule(Date.parse(originalTask.runAt), task, 30 * 60000);
       });
 
+      await delay(100);
+
```

Comment:
The additional delay caused by the buffering of `update`s allows it to fail every once in a while as the `runNow` sees the task as still running and doesn't try to run it concurrently.